### DISCUSSION TITLE
Cpu slow, fast and sustained values

### DISCUSSION THREAD

**Discussion by gotchaa1g** [seerge/g-helper#2167] [Q&A] [answered]
These vaules was 150 each but after reinstalling windows 11 vaules went to 90. Idk why, i did updated all my drivers updated my chipset too. Any help pls?
Asus Tuf A15(2023)  r7 7735hs  4050m

![Ekran görüntüsü 2024-03-06 183803](https://github.com/seerge/g-helper/assets/162473731/3add2989-72f1-4455-96c4-b715d4ab92e9)

**Comment by seerge** [comment, marked as answer]
@gotchaa1g unless Apply is checked, **this values don't have any effect.** 

Your CPU don't support power limits higher than 80W anyway, so range app shows to you is correct

**Reply by gotchaa1g** [reply]
I got it, thanks for help sir
